portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -73,6 +157,67 @@ definition {
 		}
 	}
 
+	@description = "LPS-122873 When there are some categories created for vocabulary1 (X axis), they will always have the same order during the user session"
+	@priority = "5"
+	test CategoriesForVocab1 {
```

Comment:
Change the title of this test, is not easy to know what it does. Dont refer it to concrete names